### PR TITLE
enable the videoJS player behind a flag

### PR DESCRIPTION
**Pull request by VladDubrovskis**
/cc @leggsimon @adgad @andrewgeorgiou1981

### CODE REVIEW

File: client/components/related-content/main.js [hunk 1]
```diff
@@ -45,10 +45,18 @@ module.exports = {
 
 				fetchedVideos = json.items;
 
+				let videoPlayer = '';
+				if(flags.get('brightcovePlayer')) {
+					videoPlayer = 'brightcovePlayer';
```
Comment (matthew-andrews) [inline client/components/related-content/main.js]: i think we can probably delete this

Comment (VladDubrovskis) [inline client/components/related-content/main.js]: I think there was one for brightcovePlayer in n-section or front page. will remove from there too

Comment (matthew-andrews) [inline client/components/related-content/main.js]: thank you for taking the time to tidy this up.

Comment (VladDubrovskis) [inline client/components/related-content/main.js]: No worries, Seems like i might have imagined seeing it elsewhere. Searched github and local repos and this is only place.

Shall I delete the flag from from flags api?

Comment (matthew-andrews) [inline client/components/related-content/main.js]: :) yes please

**Pull request merged**